TypeCobol/Compiler/Parser/CodeElementBuilder/CobolStatementsBuilder.cs

```diff
@@ -52,20 +52,19 @@ public CobolStatementsBuilder(CobolWordsBuilder cobolWordsBuilder, CobolExpressi
 		 // ACCEPT STATEMENT //
 		//////////////////////
 
-		internal AcceptFromInputDeviceStatement CreateAcceptDataTransferStatement(CodeElementsParser.AcceptDataTransferContext context) {
+		internal AcceptFromInputDeviceStatement CreateAcceptDataTransferStatement(ReceivingStorageArea receivingStorageArea, CodeElementsParser.FromEnvironmentContext context) {
```

Comment:
Is the name of the function still senseful?

Reply:
As it still produces the instance of the code element, i's say yes 😄.